src/main/resources/view/StudentListCard.fxml

```diff
@@ -31,7 +31,7 @@
                      <Insets left="4.0" />
                   </HBox.margin>
                </Label>
-            <Label fx:id="medicalHistory" ellipsisString="!!!" maxWidth="150.0" styleClass="cell_small_label" text="\$medicalHistory">
+            <Label fx:id="medicalHistory" ellipsisString="..." maxWidth="150.0" styleClass="cell_small_label" text="\$medicalHistory">
```

Comment:
Isnt the change to `!!!` intentional to indicate presence of medical issue?

Reply:
This is considered as a feature change, so I changed it back